apps/ledger-live-mobile/src/screens/WalletCentricSections/Allocations.tsx

```diff
@@ -59,14 +59,17 @@ const Allocations = () => {
   }, [navigation]);
 
   const distributionListFormatted = useMemo(() => {
-    const displayedCurrencies: DistributionItem[] = distribution.list.filter(
-      asset => {
+    const displayedCurrencies: DistributionItem[] = distribution.list
+      .filter(asset => {
         return (
           asset.currency.type !== "TokenCurrency" ||
           !blacklistedTokenIds.includes(asset.currency.id)
         );
-      },
-    );
+      })
+      .map(obj => {
+        const { accounts, ...other } = obj;
```

Comment:
Return the type actually expected by `RingChart` in its props (`DistributionItem`)

Reply:
If you want you can do the `filter` and `map` with just one `reduce`